spec/models/poll_answer_spec.rb

```diff
@@ -19,7 +19,7 @@
   end
 
   describe 'validation' do
-    it 'should validate pressence of answer' do
+    it 'should validate presence of answer' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.